src/components/label/label.wp.scss

```diff
@@ -40,6 +40,11 @@ $label-wp-text-color-focused:         color($colors-wp, primary) !default;
   transform-origin: left top;
 }
 
+html[dir="rtl"] .label-wp[floating] {
+  transform: translate3d(-8px, 34px, 0);
```

Comment:
The ltr translate 8px to the right, this one translate it to the left (-8px instead of 8px), am I missing something here?